daemons/ipa-slapi-plugins/ipa-pwd-extop/prepost.c

```diff
@@ -846,11 +849,13 @@ static int ipapwd_pre_mod(Slapi_PBlock *pb)
     pwdop->pwdata.target = e;
 
     /* if krb keys are being set by an external agent we assume password
-     * policies have been properly checked already, so we check them only
-     * if no krb keys are available */
+     * policies have been properly checked already. We check them only if no
+     * krb keys are available and raise error if the change is not done by a
+     * cn=Directory Manager or one of passsync managers */
     if (has_krb_keys == 0) {
         ret = ipapwd_CheckPolicy(&pwdop->pwdata);
-        if (ret) {
+        if ((pwdop->pwdata.changetype != IPA_CHANGETYPE_DSMGR) &&
+	    (ret != 0)) {
```

Comment:
My understanding is that a passwd set by DM/passsync is checked (max life and min lenght). This for krb or non_krb.
Where is it skipped for krb entries ?